### PR TITLE
Add .NET Support

### PR DESCRIPTION
**Pull request by mitchellmaler**
This fixes the issue I logged #1 at least for Dotnet.

### CODE REVIEW

**Comment by brandonkal** [comment]
Thanks for the PR @mitchellmaler!
I'm willing to merge this but could you remove the logo.png?
Also, it would probably be best to put an Apache license on the top of the files you've added so people know who to contact if they have questions (I don't write .NET much)

**Comment by mitchellmaler** [comment]
@brandonkal I added the header to the cs files and removed the logo.

File: examples/dotnet/MyStack.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// Pulumi Command Provider Node SDK
```
Comment (brandonkal) [inline examples/dotnet/MyStack.cs]: .NET SDK. Same for all other files.

Comment (mitchellmaler) [reply examples/dotnet/MyStack.cs]: That's what I get for copying it too quickly! Will update.

**Comment by mitchellmaler** [comment]
@brandonkal Should be good now.

**Pull request merged**